### PR TITLE
Document the incoming-shares:poll command

### LINKED ISSUE #856: Document occ incoming-shares:poll for update detection with federated shares
## What Needs to be Documented?
With core https://github.com/owncloud/core/pull/34903 (Add a command to poll incoming federated shares for updates)
a new occ command needs to be documented: `incoming-shares:poll`

## Where Does This Need To Be Documented?
admin_manual, occ command set

## What Type Of Content Change Is This? (Optional)
- [x] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [ ] Bug Fix to Existing Content

## Which Manual Does This Relate To? (Optional)
- [x] Admin Manual
- [ ] Developer Manual
- [ ] User Manual
- [ ] Android
- [ ] iOS
- [ ] Branded Clients
- [ ] Desktop Client
- [ ] Other

@PVince81 Pls let us know if it is correct to have this for 10.2 + no backporting to older releases
@VicDeo @jvillafanez FYI
@settermjd

### PR DESCRIPTION
**Pull request by settermjd**
This fixes #856.

💁‍♂️ **Only backport to 10.2.**

### CODE REVIEW

File: modules/admin_manual/pages/configuration/server/occ_command.adoc [hunk 1]
```diff
@@ -787,25 +787,34 @@ See xref:configuration/files/encryption_configuration.adoc[encryption configurat
...
 
-NOTE: This command is only available when the "Federation" app (`federation`) is enabled.
+For performance reasons, federated shares do not update automatically.
+Instead, federated share directories are only updated when users browse them using web UI.
+ownCloud and system administrators can use the `incoming-shares:poll` to poll federated shares for updates.
+The command polls all received federated shares. It does not require a path.
+
+[source,console]
+----
+sudo -u www-data ./occ incoming-shares:poll
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/server/occ_command.adoc]: We should use `sudo -u www-data php occ`, same for above.
(Might be that there are some leftovers at other places too, but that could be corrected in an other PR)

File: modules/admin_manual/pages/configuration/server/occ_command.adoc [hunk 2]
```diff
@@ -787,25 +787,34 @@ See xref:configuration/files/encryption_configuration.adoc[encryption configurat
...
+NOTE: This command is only available when the "Federation" app (`federation`) is enabled.
 
-....
-sudo -u www-data php occ federation:sync-addressbooks
-....
+=== Poll Incoming Federated Shares For Updates
 
-NOTE: This command is only available when the "Federation" app (`federation`) is enabled.
+For performance reasons, federated shares do not update automatically.
+Instead, federated share directories are only updated when users browse them using web UI.
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/server/occ_command.adoc]: using the webUI

File: modules/admin_manual/pages/configuration/server/occ_command.adoc [hunk 3]
```diff
@@ -787,25 +787,34 @@ See xref:configuration/files/encryption_configuration.adoc[encryption configurat
...
+ownCloud and system administrators can use the `incoming-shares:poll` to poll federated shares for updates.
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/server/occ_command.adoc]: either:
using the incoming-shares:poll command to poll...
or just
using incoming-shares:poll to poll...

File: modules/admin_manual/pages/configuration/server/occ_command.adoc [hunk 4]
```diff
@@ -787,25 +787,34 @@ See xref:configuration/files/encryption_configuration.adoc[encryption configurat
...
+The command polls all received federated shares. It does not require a path.
+
+[source,console]
+----
+sudo -u www-data php occ incoming-shares:poll
+----
+
+IMPORTANT: We recommend running the command no more than once every twelve hours. 
+However, for instances with low federated shares amount, the command can be run once every two hours.
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/server/occ_command.adoc]: with a low number of federated shares, ...

**Comment by VicDeo** [comment]
@settermjd probably it is worth mentioning that `incoming-shares:poll` is needed only if one wants to update federated shares with a Desktop client on a regular basis.

**Comment by settermjd** [comment]
@VicDeo - changes made.

File: modules/admin_manual/pages/configuration/server/occ_command.adoc [hunk 5]
```diff
@@ -787,25 +787,37 @@ See xref:configuration/files/encryption_configuration.adoc[encryption configurat
...
-synchronize federated servers:
+NOTE: This command is only available when the "Federation" app (`federation`) is enabled.
 
-....
-sudo -u www-data php occ federation:sync-addressbooks
-....
+=== Poll Incoming Federated Shares For Updates
 
-NOTE: This command is only available when the "Federation" app (`federation`) is enabled.
+This command must be used if federated shares are being updated with a Desktop client or on a regular basis.
```
Comment (VicDeo) [inline modules/admin_manual/pages/configuration/server/occ_command.adoc]: @phil-davis could you please proof-read this sentence? I'm not sure whether it is clear enough.

Comment (phil-davis) [reply modules/admin_manual/pages/configuration/server/occ_command.adoc]: Maybe:
This command must be used if federated shares are being referenced by Desktop clients but not regularly accessed via the webUI.

Comment (settermjd) [reply modules/admin_manual/pages/configuration/server/occ_command.adoc]: @phil-davis, that works perfectly for me. Thanks.

Comment (settermjd) [reply modules/admin_manual/pages/configuration/server/occ_command.adoc]: @phil-davis, @VicDeo the sentence has been updated.

**Comment by VicDeo** [comment]
@settermjd Thanks, it looks perfect to me now.

**Pull request merged**